modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/exportimport/data/handler/MBCategoryStagedModelDataHandler.java

```diff
@@ -149,14 +149,21 @@ protected void doImportStagedModel(
 		MBCategory importedCategory = null;
 
 		if (portletDataContext.isDataStrategyMirror()) {
-			MBCategory existingCategory = fetchStagedModelByUuidAndGroupId(
-				category.getUuid(), portletDataContext.getScopeGroupId());
+			MBCategory existingCategory =
+				_mbCategoryLocalService.fetchMBCategoryByExternalReferenceCode(
```

Comment:
What is the reason to delete the log? I follow the structure of the classes and I didn't see a similar log in the api. Am i missing something? This log is useful when we are using staging and export/import process.

Reply:
Well, so I don't fully understand Brian suggestion. Maybe he wants me to use get+try+catch ?

Reply:
BTW, I copied this log structure from another class.

Reply:
Yes, you have to use getMBCategoryByExternalReferenceCode and inside it there is a log similiar when we are calling to findByERC_G of the persistence and of corse, you will have to catch the exception